framework/components/ecl-carousels/ecl-carousels.twig

```diff
@@ -0,0 +1,91 @@
+{#
+
+Usage:
+
+  - css_classes: (string)
+    Possible set of classes in addition to `ecl-carousel` wrapper.
+
+  - extra_attributes: (iterable)
+    Conventional map of possible extra attributes for the wrapper.
+
+  - heading_attributes: (iterable)
+    Conventional map of possible extra attributes for the header.
+
+  - heading_title: (string)
+    Optional title of the heading. If skipped, heading will not be shown.
+
+  - carousel_images (iterable)
+    A list of `@ec-europa/ecl-images` components.
+
+#}
+
+
+{# Internal properties #}
+
+{% set _css_classes = 'ecl-carousel' %}
+{% set _extra_attributes = '' %}
+{% set _heading_attributes = '' %}
```

Comment:
Let's discuss on slack if the heading would be an obligatory element and it's defaults.